app/views/events/show.html.erb

```diff
@@ -100,15 +100,7 @@
                     <%= link_to invoice.display_title, invoice %>
                 </td>
                 <td>
-                  <% if can? :status, invoice %>
```

Comment:
To be thorough, leave this `if` statement. That way if permissions get changed in the future the box will still be show/hidden properly and it won't break the `:status` permission when used.